core/server/api/v2/utils/serializers/input/posts.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     all(apiConfig, frame) {
         debug('all');
 
-        if (!_.get(frame, 'options.context.user') && _.get(frame, 'options.context.client_id')) {
+        if (!_.get(frame, 'options.context.user') && _.get(frame, 'options.context.api_key_id')) {
```

Comment:
I think the http wrapper no longer has to forward the client_id, because it's always null
See shared/http 👍

Reply:
Yup that's right :) was gonna do in a separate PR - can add it here though?

Reply:
Here is fine 👍